CodeEdit/Breadcrumbs/BreadcrumbsComponent.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BreadcrumbsComponent.swift
+//  CodeEdit
+//
+//  Created by Lukas Pistrol on 18.03.22.
+//
+
+import SwiftUI
+
+struct BreadcrumbsComponent: View {
```

Comment:
nit: it seems that there are some identation issues below

Reply:
Weird in my Xcode everything seems fine. Nevertheless I `ctrl+I`ed it now.

Reply:
ah might be something weird with github then!